contracts/src/test/7683/T1XChainReaderBaseTestSetup.sol

```diff
@@ -64,6 +64,7 @@ contract T1XChainReaderBaseTestSetup is BaseTest {
             destinationDomain: destination,
             destinationSettler: address(l2T1ERC7683).addressToBytes32(),
             fillDeadline: uint32(block.timestamp + 100),
+            closedAuction: false,
```

Comment:
What is the significance of this flag? Would be good to have some documentation on it

Reply:
It will be true if we run a closed auction offchain on our auction backend and false if the intent is free to be filled by anyone. But maybe there could be a better name ?